test/model_test.dart

```diff
@@ -1260,12 +1260,12 @@ String topLevelFunction(int param1, bool param2, Cool coolBeans,
     });
 
     test('COLOR_ORANGE has correct value', () {
-      expect(orangeConstant.constantValue, "'orange'");
+      expect(orangeConstant.constantValue, "&#39;orange&#39;");
```

Comment:
It might be worth adding a test for documentation of a generic, since that was the failing case in flutter.  Doesn't look like current testing covers Foo<Bar,Baz>